core/src/main/java/novoda/lib/sqliteprovider/provider/SQLiteContentProviderImpl.java

```diff
@@ -96,14 +95,12 @@ private Uri insertSilently(Uri uri, ContentValues values) {
     protected int updateInTransaction(Uri uri, ContentValues values, String selection, String[] selectionArgs) {
         ContentValues insertValues = (values != null) ? new ContentValues(values) : new ContentValues();
```

Comment:
i know this was already here but afaik we don't need to do this and can actually pass the `null` to `update()`